container/Makefile

```diff
@@ -1,47 +1,65 @@
-POD_NAME = faf-pod
-VOLUME_NAME = faf-db-volume
+DB_IMAGE=abrt/faf-db
+LOCAL_IMAGE=abrt/faf-local
+IMAGE=abrt/faf
+
+DB_CONTAINER=faf-db
+CONTAINER=faf
+
+POD = faf-pod
+
+VOLUME = faf-db
+
 DB_PASS="NotSoSecretPassword"
+ENVIRONMENT = \
+	"-e PGHOST=localhost" \
+	"-e PGUSER=faf" \
+	"-e PGPASSWORD=scrt" \
+	"-e PGPORT=5432" \
+	"-e PGDATABASE=faf" \
+	"-e RDSBROKER=redis://faf-redis:6379/0" \
+	"-e RDSBACKEND=redis://faf-redis:6379/0"
+
+build: Dockerfile
+	podman build --file=$< --tag=$(IMAGE) ../
 
-build:
-	podman build -t faf-image -f Dockerfile ../
-	podman tag faf-image abrt/faf-image
+build_local: Dockerfile
+	podman build --build-arg=titoargs=--test --file=$< --format=docker --tag=$(LOCAL_IMAGE) ../
 
-build_local:
-	podman build -t faf-image-local -f Dockerfile_local ../
+build_db: Dockerfile_db
+	podman build --file=$< --tag=$(DB_IMAGE) ../
 
-build_db:
-	podman build -t postgres-semver -f Dockerfile_db ../
-	podman tag postgres-semver abrt/postgres-semver
+.PHONY: pod
+pod:
+	podman pod create -p 5432:5432 -p 6379:6379 -p 8080:8080 --name $(POD) || :
 
 run:
-	podman run --pod $(POD_NAME) --name faf -dit -e PGHOST=localhost -e PGUSER=faf -e PGPASSWORD=scrt -e PGPORT=5432 -e PGDATABASE=faf -e RDSBROKER=redis://faf-redis:6379/0 -e RDSBACKEND=redis://faf-redis:6379/0 abrt/faf-image
+	podman run --detach --interactive --pod $(POD) --name $(CONTAINER) --tty $(ENVIRONMENT) $(IMAGE)
 
-run_local:
-	podman run --pod $(POD_NAME) --name faf -dit -e PGHOST=localhost -e PGUSER=faf -e PGPASSWORD=scrt -e PGPORT=5432 -e PGDATABASE=faf -e RDSBROKER=redis://faf-redis:6379/0 -e RDSBACKEND=redis://faf-redis:6379/0 faf-image-local
+run_local: pod
+	podman run --detach --interactive --pod $(POD) --name $(CONTAINER) --tty $(ENVIRONMENT) $(LOCAL_IMAGE)
```

Comment:
Right, so, for development, this will be really slow. Not sure what would be the better strategy.